jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/DefaultServlet.java

```diff
@@ -186,19 +185,25 @@ public void init() throws ServletException
 
         _resourceService.setAcceptRanges(getInitBoolean("acceptRanges", _resourceService.isAcceptRanges()));
         _resourceService.setDirAllowed(getInitBoolean("dirAllowed", _resourceService.isDirAllowed()));
-        _resourceService.setRedirectWelcome(getInitBoolean("redirectWelcome", _resourceService.isRedirectWelcome()));
+        boolean redirectWelcome = getInitBoolean("redirectWelcome", false);
+        _resourceService.setWelcomeMode(redirectWelcome ? ResourceService.WelcomeMode.REDIRECT : ResourceService.WelcomeMode.SERVE);
```

Comment:
So how would I set `WelcomeMode.REHANDLE`? Or is this inappropriate in the servlet environment - if so, a javadoc to that effect would be good.

Reply:
No `REHANDLE` in Servlets, it's equivalent to `SERVE`.
As this parameter is, and always was, a `boolean`, I don't think there is need to mention `REHANDLE` which is not applicable to Servlets anyway.
To be clear, I don't think it's worth mentioning a 3rd mode, just to say that you cannot use it.
Plus the parameter name would need to be changed, as it contains "redirect".